components/colorwheel/index.css

```diff
@@ -1,48 +1,72 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
 .spectrum-ColorWheel {
-  /* todo: fix this in DNA */
-  --spectrum-colorwheel-focus-ring-size: var(--spectrum-global-dimension-static-size-25);
-  --spectrum-colorwheel-border-radius: 100%;
-  --spectrum-colorwheel-width: calc(var(--spectrum-global-dimension-size-125) * 16);
   --spectrum-colorwheel-height: var(--spectrum-colorwheel-width);
-}
-
-%spectrum-ColorControl-handle--focused {
-  /* Bigger handle when focused */
-  width: calc(var(--spectrum-colorhandle-size) * 2);
-  height: calc(var(--spectrum-colorhandle-size) * 2);
+  --spectrum-colorwheel-border-color: var(--spectrum-transparent-black-200);
+  --spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
+  --spectrum-colorwheel-fill-color-disabled: var(--spectrum-disabled-background-color);
+  --spectrum-colorwheel-track-width: var(--spectrum-color-control-track-width);
+  --spectrum-colorwheel-colorarea-margin: var(--spectrum-color-wheel-color-area-margin);
+
+  --spectrum-colorwheel-colorhandle-position: calc((var(--spectrum-colorwheel-width) / 2) - (var(--spectrum-colorwheel-track-width) / 2));
+
+  .spectrum--medium & {
+    --spectrum-colorwheel-width: 192px; /* TODO replace with incoming color-wheel-width token */
+    --spectrum-colorwheel-min-width: 175px; /* TODO replace with incoming color-wheel-minimum-width token */
```

Comment:
This is now available in the tokens package as: `--spectrum-color-wheel-minimum-width`, but as noted above, for consistency, we should probably do something like:

```css
--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
```

Reply:
tokens for width and min width have been updated!